solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -135,17 +137,25 @@ contract WalletRegistry is Ownable {
         maliciousDkgResultNotificationRewardMultiplier = 100;
 
         dkg.init(_sortitionPool, _dkgValidator);
+        dkg.setSeedTimeout(1440); // ~6h assuming 15s block time // TODO: Verify value
         dkg.setResultChallengePeriodLength(11520); // ~48h assuming 15s block time
         dkg.setResultSubmissionTimeout(100 * 20); // TODO: Verify value
         dkg.setSubmitterPrecedencePeriodLength(20); // TODO: Verify value
     }
 
+    // TODO: Update to governable params
+    function updateRandomBeacon(IRandomBeacon _newRandomBeacon) external {
+        randomBeacon = _newRandomBeacon;
+    }
+
     // TODO: Update to governable params
     function updateDkgParams(
+        uint256 newSeedTimeout,
         uint256 newResultChallengePeriodLength,
         uint256 newResultSubmissionTimeout,
         uint256 newSubmitterPrecedencePeriodLength
     ) external {
+        dkg.setSeedTimeout(newSeedTimeout);
```

Comment:
Can we merge first 2800 with the gov params? This entire function will be gone and replaced by the ownable `updateDkgParams`. Everytime a new gov params is introduced it produces conflicts with 2800 PR which creates double work, here and there.